### ISSUE TITLE
ExportToDatabase to be like ExportToSpreadsheet.  Single group databases, and object measurement limitations.

### ISSUE THREAD

**Issue by leeleavitt** [CellProfiler/CellProfiler#2611]
Hello all.  I have embarked on using CP for time series analysis in conjunction with multiple cell labels.  I am working with hour long calcium imaging data with over 2000 objects per experiment.  We have an arsenal to work through this data once it is exported, which is why i am just trying to get an automated system for this export.

I have run into limitations with ExportToSpreadsheet (cannot produce information for more than 5000 objects), although this is my favorite module i am forced to limit my objects and this leads to other more frustrating problems.

I have learned that with this amount of data I need to work in databases.  The problem is my experiments are so large that i need a database per experiment so that i can work on each separately.  I am working with between 4 to 6 experiments a day.  Unfortunate for me, ExportToDatabase produces a single database for all group analyzed.  I am able to merge databases later, but i really need a single experiment/group to be exported to a single database for now.

Another thing that i like about ExportToSpreadSheet is the option to limit the measurements.  Do not get me wrong I am happy to see all the data initially, but since my experiments are so large, i will eventually need to limit my output to a few object measurements.

Thank you for your consideration.  Cellprofiler is a fantastic program, and is pushing the limits on image analysis.

**Comment by mcquin** [comment]
I agree that it would be nice to have consistency across the Export modules. They really should be identical up to the decision on where to store the data.

> The problem is my experiments are so large that i need a database per experiment so that i can work on each separately.
>  I am able to merge databases later, but i really need a single experiment/group to be exported to a single database for now.

I think there are other ways to achieve this functionality. Have you tried running one instance of CellProfiler per group?

**Comment by leeleavitt** [comment]
Hey,  I have, but it really becomes a pain when i have to update each import module,  then save that data as a csv, then save cellprofiler pipelines for each experiment.  It becomes very complicated, and hard for me to sell to my group.

**Comment by mcquin** [comment]
Can you make use of [this suggestion on the forum](http://forum.cellprofiler.org/t/exporttodatabase-output-file-location-error/4395/17) for a more automated approach?

**Comment by mcquin** [comment]
@leeleavitt We will welcome a submission to CellProfiler with the behavior you describe (adding an option to save one database per group). :)

**Comment by leeleavitt** [comment]
Great.  Is there anything i need to do?
